run_lmod_cache.py

```diff
@@ -133,10 +145,8 @@ def main():
                 opts.critical("Lmod cache update failed")
 
         opts.log.info("Checking the Lmod cache freshness")
-        timestamp = find_oldest_cache(opts.options.module_basedir, archs=opts.options.architecture)
-
-        # give a warning when the cache is older then --freshness-threshold
-        if (time.time() - timestamp) > opts.options.freshness_threshold * 60:
```

Comment:
now I look at the code, we actually already have this more or less? We only need to run with `--disable-create-cache'?